src/dependencies.js

```diff
@@ -105,7 +105,7 @@ const isNextOnNetlify = function(dependency) {
 // we tree shake direct local requires to bundle only relevant code
 // we don't tree shake external dependencies since they can break our tree shaking logic by:
 // 1. doing dynamic requires (e.g. require(dynamicPath)
-// 2. reading files (e.g. fs.readFileSync(someNonJsFile)
+// 2. reading files (e.g. fs.readFileSync(someNonJsFile))
```

Comment:
In the comment above too! :)

Reply:
🤦

Reply:
done in https://github.com/netlify/zip-it-and-ship-it/pull/267/commits/221a88552454c954bcfb8c33cc88f3acb82c7194